frame/evm/src/lib.rs

```diff
@@ -677,6 +678,35 @@ pub mod pallet {
 
 	#[pallet::storage]
 	pub type Suicided<T: Config> = StorageMap<_, Blake2_128Concat, H160, (), OptionQuery>;
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_initialize(_: BlockNumberFor<T>) -> Weight {
+			let mut total_weight = Weight::zero();
+
+			// Do dummy read to populate the pov with the intermediates nodes,
```

Comment:
It's not dummy as it will run in production.
I guess it's more about whitelisting some storage keys, if so can you explain that in comments source code?

Reply:
I meant dummy as in the result of the read is not used. But I removed this part to avoid confusion.